### ISSUE TITLE
Can't delete newly-created functions

### ISSUE THREAD

**Issue by maxknivets** [progrium/vizgo#57]
![ezgif com-video-to-gif(5)](https://user-images.githubusercontent.com/34922382/84525803-eacc4d00-ace4-11ea-8dcb-626911fbc5b3.gif)

**Comment by maxknivets** [comment]
This only happens whenever a function is created. Other declarations work fine.

**Comment by progrium** [comment]
should also be fixed in 504dd0b8fb642fe7416e9f7deffcca365cde153b though i think i ran into a race condition clicking trash for a function not selected where it is deleted but not shown and clicking again will crash. thats a sep issue if real

**Comment by maxknivets** [comment]
I've just tried the changes out and it doesn't seem to work for me. 
![ezgif com-video-to-gif(6)](https://user-images.githubusercontent.com/34922382/84532487-fe7cb100-acee-11ea-8454-089f58cad5e6.gif)

**Comment by maxknivets** [comment]
Also happens whenever there are multiple functions at once:
![ezgif com-video-to-gif(7)](https://user-images.githubusercontent.com/34922382/84532783-806cda00-acef-11ea-95cc-8818f74f8504.gif)

**Comment by progrium** [comment]
yes like i said, it's a separate issue. click the function panel first
